Mediator.h

```diff
@@ -10,13 +10,12 @@ template <typename Item> class Mediator
 {
 public:
   
-  Mediator( int nItems )
+  Mediator(int nItems ):N(nItems)
```

Comment:
What does the `:N(nItems)` do? I haven't seen that syntax before (granted, my C++ is very rusty...)